src/main/webapp/CollapsiblePanel/SampleDataCohortsCollapsibleHeader.js

```diff
@@ -80,13 +80,13 @@ export const SampleDataCohortsCollapsibleHeader = hh(class SampleDataCohortsColl
         div({className: 'panel-title'}, [
           div({className: 'cta-container'}, [
             button({
-              isRendered: status === STATUS.pending || isEmpty(status) && component.isAdmin,
+              isRendered: component.isAdmin && status === STATUS.pending || isEmpty(status),
               className: 'btn btn-default btn-sm confirmationModal',
               style: styles.pointer.auto,
               onClick: (e) => approveHandler(e, projectKey)
             },['Approve']),
             button({
-              isRendered: status === STATUS.pending || isEmpty(status) && component.isAdmin,
+              isRendered: component.isAdmin && status === STATUS.pending || isEmpty(status),
```

Comment:
I really wish I had caught this earlier. Having three booleans here is very difficult to reason about. From a js repl:
```
true && true || true => true
true && true || false => true
true && false || true => true
false && true || true => true
true && false || false => false
false && true || false => false
false && false || false => false
```
I suggest parentheses or extracting to a function that is easier to read/reason about is a better way to make this check.

Reply:
:+1: Yes, totally agree.